docs/usage/puppet.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
Do we really need this new page? Can't we write this to the manager readme instead?

Reply:
moved in manager readme.md